src/fiatconnect/fiatAccountSchemas/mobileMoney.test.ts

```diff
@@ -0,0 +1,31 @@
+import { FiatAccountType } from '@fiatconnect/fiatconnect-types'
+import { getMobileMoneySchema } from 'src/fiatconnect/fiatAccountSchemas/mobileMoney'
+import { FormFieldParam } from 'src/fiatconnect/fiatAccountSchemas/types'
+
+const validPhoneNumber = '+123456789'
+const invalidPhoneNumber = '+1234567890123456789' // too long
+
+describe(getMobileMoneySchema, () => {
+  describe('mobileMoney.validate', () => {
+    const { validate, format } = getMobileMoneySchema({
+      country: 'US',
+      fiatAccountType: FiatAccountType.MobileMoney,
+    }).mobile as FormFieldParam
+    it('fails on invalid input', () => {
+      expect(validate(invalidPhoneNumber).isValid).toBeFalsy()
+      // failure returns default error message
+      expect(validate(invalidPhoneNumber).errorMessage).toEqual(
+        'fiatAccountSchema.mobileMoney.mobile.errorMessage'
+      )
+    })
+    it('succeeds on valid input', () => {
+      expect(validate(validPhoneNumber).isValid).toBeTruthy()
+      expect(validate(validPhoneNumber).errorMessage).toBeUndefined()
+    })
+    it('formatter adds "+" symbol', () => {
```

Comment:
nit: either move this to a different describe since it doesn't test `validate` or just drop the `mobileMoney.validate` describe and rename above tests to include "validate". e.g., `validate fails/succeeds on`